shared/chat/conversations-list/container.js

```diff
@@ -20,9 +21,10 @@ class ConversationListContainer extends Component {
 export default connect(
   (state: TypedState, {routeSelected}) => ({
     conversationUnreadCounts: state.chat.get('conversationUnreadCounts'),
-    inbox: state.chat.get('inbox').filter(i => !i.isEmpty || i.youCreated),
+    inbox: state.chat.get('inbox').filter(i => !i.isEmpty || i.youCreated)
```

Comment:
:sheep: This filtering logic is getting pretty intense... probably time to move it out of this connect and into a helper